en/sim_gazebo_gz/README.md

```diff
@@ -98,11 +107,23 @@ In particular, it is possible to:
 
 These scenarios are managed by setting the appropriate environment variables.
 
+:::note
+The environment variables described below can be used as a prefix to the `make px4_sitl gz*` command as well as for starting PX4 SITL from a prebuilt binary.
+
+These will override any settings defined by the make command itself.
+For example, this following command will start the `gz_rc_cessna` model rather than the `gz_x500` model.
```

Comment:
@frede791 This is an assumption - is it correct that the environment variables override make settings as I have indicated? If not, what's the rule?

Reply:
This is correct: `PX4_GZ_MODEL` overrides make settings. However, the model itself does not actually load into Gazebo. It throws: `Error finding file [gz_rc_cessna/model.sdf].`. This even happens when you provide the same model for the make settings and `PX4_GZ_MODEL`. 
Why this happens I don't know but I'll investigate it further. Based on this, however, I wouldn't advertise this as a way of starting Gazebo.